assets/sass/05-blocks/quote/_editor.scss

```diff
@@ -1,19 +1,52 @@
 .wp-block-quote {
-	border-left-color: var(--quote--border-color);
-	border-left-width: var(--quote--border-width);
+	border: none;
 	margin: var(--global--spacing-vertical) 0;
 	padding-left: var(--global--spacing-horizontal);
 
+	&::before {
+		color: currentColor;
+		content: "\201C";
+		display: block;
+		font-size: 3rem;
+		font-weight: 700;
+		line-height: 1;
+		position: absolute;
+		top: 4px;
```

Comment:
I'm not sure if it's just my fonts/browser, but this lines up better with `top: 0`

![Screen Shot 2020-10-15 at 11 23 36 AM](https://user-images.githubusercontent.com/541093/96150937-e1dcc980-0ed8-11eb-90e5-a30ece317c4b.png)